### PR TITLE
e2e test helper script

### PR DESCRIPTION
**Pull request by cgchinmay**
### Issue
N/A

### Description
Added helper script to deploy load balancer controller and trigger e2e tests against a given cluster.
Needed script to run tests on prow 

### Checklist
Manually tested

### CODE REVIEW

File: test/e2e/run.sh [hunk 1]
```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+# This script runs e2e tests on the AWS Load Balancer Controller
+
+set -e
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+echo "Running AppMeshController integration test with the following variables
```
Comment (kishorj) [inline test/e2e/run.sh:8]: "Load Balancer Controller" :)

File: test/e2e/run.sh [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+KUBE CONFIG: $KUBE_CONFIG_PATH
```
Comment (kishorj) [inline test/e2e/run.sh:9]: any reason to not use KUBECONFIG?

Comment (cgchinmay) [reply test/e2e/run.sh:9]: We do specify $KUBE_CONFIG_PATH when triggering ginkgo tests . This KUBE CONFIG is only for logging information.

File: test/e2e/run-canary-test.sh [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+else
+  CLUSTER_INFO=$($GET_CLUSTER_INFO_CMD --endpoint $ENDPOINT)
+fi
+
+VPC_ID=$(echo $CLUSTER_INFO | jq -r '.cluster.resourcesVpcConfig.vpcId')
+ACCOUNT_ID=$(aws sts get-caller-identity | jq -r '.Account')
+
+echo "VPC ID: $VPC_ID"
+
+eksctl utils associate-iam-oidc-provider \
```
Comment (M00nF1sh) [inline test/e2e/run-canary-test.sh:31]: can this be done in the centralized image by default?

File: test/e2e/run-canary-test.sh [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+echo "Update helm repo eks"
+helm repo add eks https://aws.github.io/eks-charts
+
+helm repo update
+
+echo "Install TargetGroupBinding CRDs"
+kubectl apply -k "github.com/aws/eks-charts/stable/aws-load-balancer-controller//crds?ref=master"
+
+echo "Install aws-load-balacner-controller"
```
Comment (M00nF1sh) [inline test/e2e/run-canary-test.sh:60]: s/balacner/balancer/

**Review by M00nF1sh — APPROVED** [review]
/lgtm

File: test/e2e/run-canary-test.sh (hunk 3 above)
Comment (cgchinmay) [reply test/e2e/run-canary-test.sh:31]: We have similar script for other components as well and those scripts are reused to run tests, we can keep this in default image but then other component scripts will have to be modified as well. So kept this code here to have consistent pattern.

**Pull request merged**